packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -407,6 +412,13 @@ class ObjectInspector extends Component {
             onClick: onLabelClick
               ? event => {
                 event.stopPropagation();
+
+                const selection = getSelection();
+                // If the user selected text, bail out.
+                if (selection && selection.type === "Range") {
+                  return;
+                }
+
```

Comment:
nit: you might introduce a new `this.getSelection` method that wraps the logic.

Reply:
good idea, I created a util for that (`documentHasSelection`)